peekingduck/pipeline/nodes/input/recorded.py

```diff
@@ -24,7 +24,34 @@
 
 # pylint: disable=R0902
 class Node(AbstractNode):
-    """Node to receive videos/image as inputs."""
+    """Node to receive videos/image as inputs.
+
+        Inputs:
+            None
+
+        Outputs:
+            |img|
+
+            |pipeline_end|
+
+            |filename|
+
+            |saved_video_fps|
+
+        Configs:
+            resize (:obj:`Dict`): **default = { do_resizing: False, width: 1280, height: 720 }**
+
+                User defined dimension of extracted image frame
+
+            input_dir (:obj: `str`): **default = 'PeekingDuck/data/input'**
+
+                The directory to look for recorded video files and images
+
+            mirror_image (:obj:`bool`): **default = False**
+
+                User's choice to set extracted image frame as mirror image of input stream
```

Comment:
use the same definition as `live.py`

Reply:
i will stick to "Boolean to set extracted image frame as mirror image of input stream" for both live and recorded